.dockstore.yml

```diff
@@ -48,72 +48,76 @@ workflows:
      filters:
        branches:
          - master
-   - name: GvsPrepareCallset
+   - name: funcotator
      subclass: WDL
-     primaryDescriptorPath: /scripts/variantstore/wdl/GvsPrepareCallset.wdl
+     primaryDescriptorPath: /scripts/funcotator_wdl/funcotator.wdl
      testParameterFiles:
-       - /scripts/variantstore/wdl/GvsPrepareCallset.example.inputs.json
+     - /scripts/funcotator_wdl/funcotator.json
      filters:
        branches:
          - master
-         - ah_var_store
-   - name: ImportGenomes
+   - name: GvsCreateFilterSet
      subclass: WDL
-     primaryDescriptorPath: /scripts/variantstore/wdl/ImportGenomes.wdl
+     primaryDescriptorPath: /scripts/variantstore/wdl/GvsCreateFilterSet.wdl
      testParameterFiles:
-       - /scripts/variantstore/wdl/ImportGenomes.example.inputs.json
+       - /scripts/variantstore/wdl/GvsCreateFilterSet.example.inputs.json
      filters:
        branches:
          - master
          - ah_var_store
-   - name: funcotator
+         - mmt_SA_support_and_wdl_renaming
```

Comment:
TODO: remove branch references before merging

Reply:
actually will wait for a future PR so it doesn't break for anyone using these WDLs